### PR TITLE
Add page to docs for description syncing

### PR DESCRIPTION
**Pull request by farkmarnum**
[preview](https://glean-documentation-mhkac38l1-glean-docs.vercel.app/docs/data-modeling/description-syncing)

### CODE REVIEW

**Comment by glean-calder** [comment]
Going to pass this off to @vndrewlee for the final say!

File: pages/docs/data-modeling/description-syncing.mdx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Glean will automatically import that description when you create a Data Model for that table:
+
+<img
+  src="/description_syncing/glean-attribute-card-with-description.png"
+  alt="Glean automatically imports the description."
+  style={{ padding: "20px 0" }}
+/>
+
+In addition, you can import descriptions explicitly using the Import Descriptions feature:
```
Comment (glean-calder) [inline pages/docs/data-modeling/description-syncing.mdx:30]: Nit: casing of `Import Descriptions` doesn't match the UI

Comment (glean-calder) [inline pages/docs/data-modeling/description-syncing.mdx:30]: Nit: we should probably wrap `Import descriptions` in a `<code>` block or something of the like

File: pages/docs/data-modeling/description-syncing.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import { Callout } from "nextra-theme-docs";
+
+# Description Syncing
+
+If you've added descriptions to your columns via column comments/descriptions or
+DBT `persist-docs`, Glean will automatically import those descriptions when you
+create or update a Data Model.
+
+In addition, if the comments/descriptions in your table change, you can import
+the latest values using the "Import Descriptions" feature in the Model Builder.
```
Comment (glean-calder) [inline pages/docs/data-modeling/description-syncing.mdx:10]: Nit: we use quotes here but not below when referencing the feature. We should reference it consistently (and perhaps using something stronger than quotes such as a `<code>` tag)

File: pages/docs/data-modeling/description-syncing.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { Callout } from "nextra-theme-docs";
+
+# Description Syncing
+
+If you've added descriptions to your columns via column comments/descriptions or
+DBT `persist-docs`, Glean will automatically import those descriptions when you
+create or update a Data Model.
```
Comment (vndrewlee) [inline pages/docs/data-modeling/description-syncing.mdx:7]: is it true that whenever i update a model, the descriptions get synced? i think its always an explicit user action?

**Comment by vndrewlee** [comment]
added a comment about a sentence i found confusing, but otherwise lgtm

**Review by vndrewlee — APPROVED** [review]

File: pages/docs/data-modeling/description-syncing.mdx (hunk 3 above)
Comment (farkmarnum) [reply pages/docs/data-modeling/description-syncing.mdx:7]: good catch!

**Pull request merged**